cmd/completion.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// completionCmd represents the completion command
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script",
+	Long: `To load completions:
+
+
```

Comment:
we should probably add some examples. For example since this prints to stdout seems like we have to do something like this for it to work

```
ssullivan@data ~/golang/src/github.com/liquidweb/liquidweb-cli $ lw completion bash >> ~/.bashrc 
ssullivan@data ~/golang/src/github.com/liquidweb/liquidweb-cli $ source ~/.bashrc 
```